Configs/AuthenticatorSyncShared.xcconfig

```diff
@@ -0,0 +1,4 @@
+#include "./Common.xcconfig"
+#include? "./Local.xcconfig"
+
+PRODUCT_BUNDLE_IDENTIFIER = $(ORGANIZATION_IDENTIFIER).AuthenticatorSyncShared
```

Comment:
🎨 All of our other bundle IDs are kebab-cased as near as I can tell; it probably makes sense for us to continue that

Reply:
Fixed in next commit. 👍